doc/userguide/rules/tag.rst

```diff
@@ -0,0 +1,133 @@
+Tag
+===
+
+The `tag` keyword allows tagging of the current and future packets.
+
+Tagged packets can be logged in `EVE` and conditional PCAP logging.
+
+Tagging is limited to a scope: `host` or `session` (flow). When using `host` a
+direction can be specified: `src` or `dst`. Tagging will then occur based on the
+`src` or `dst` IP address of the packet generating the alert.
+
+Tagging is further controlled by count: `packets`, `bytes` or `seconds`. If the
+count is ommited built-in defaults will be used:
+
+- for `session`: 256 packets
+- for `host`: 256 packets for the destination IP of the packet triggering the alert
+
+The `tag` keyword can appear multiple times in a rule.
+
+Syntax
+~~~~~~
+
+::
+
+    tag:<scope>[,<count>, <metric>[,<direction>]];
+
+Values for `scope`: `session` and `host`
+Values for `metric`: `packets`, `bytes`, `seconds`
+Values for `direction`: `src` and `dst`
+
+.. note:: When the scope is "host", count and metric need to be specified to change the
+   direction.
```

Comment:
This not is a bit difficult to understand. Maybe just adding something like "as illustrated in the last two examples below" would help.

Reply:
what I want to say is:
`tag:session;` ok
`tag:host;` ok
`tag:host,100,packets,src;` ok
`tag:host,src;` NOT ok <- want to make clear to specify `src` you need the count and metrics too.

Reply:
I understood it, after reading and checking the examples a few times. But at first, it was hard to get. Maybe if we move the note to right below the examples, and highlight this aspect on them?

Reply:
reworded in #9801